src/libraries/System.Security.Cryptography/tests/X509Certificates/ChainPolicyTests.cs

```diff
@@ -0,0 +1,143 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Formats.Asn1;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public static class ChainPolicyTests
+    {
+        [Fact]
+        public static void DefaultCtorState()
+        {
+            X509ChainPolicy policy = new X509ChainPolicy();
+            AssertDefaultState(policy);
+        }
+
+        [Fact]
+        public static void ResetAfterChanges()
+        {
+            using (X509Certificate2 cert = new X509Certificate2(SelfSigned1PemBytes))
+            {
+                X509ChainPolicy policy = new X509ChainPolicy();
+                policy.CertificatePolicy.Add(new Oid("0.0"));
```

Comment:
Kind of a nit but these aren't valid OIDs. Are you explicitly testing that? If not, would it make sense to use something more realistic? (Same Q for the rest of the tests)

Reply:
I believe "0.0" is valid OID per the [ITU-T X.660 (07/2011)](https://www.itu.int/rec/T-REC-X.660-201107-I)

Section 7.3
> The integer-valued Unicode label shall contain no characters other than those in the range from the DIGIT
> ZERO character to the DIGIT NINE character and shall not commence with a DIGIT ZERO character, unless it has
> only a single character and the primary integer value of the arc is zero.

Reply:
Ah, okay, I am thinking of how SNMP defines OIDs and the "null" OID is not permitted.

Reply:
They aren't valid in context, but 0.0 is `{itu-t(0) recommendation(0)}` and 1.1 is `{iso(1) registration-authority(1)}`.  Since nothing uses the policy object here it's just "make the collection be non-empty"